src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -37,13 +36,18 @@
 @Inheritance(strategy = InheritanceType.SINGLE_TABLE)
 @DiscriminatorColumn(name = "discriminator", discriminatorType = DiscriminatorType.STRING)
 @DiscriminatorValue(value = "E")
-@DiscriminatorOptions(force = true)
```

Comment:
The annotation was not necessary at all, this was a misunderstanding when I introduced it. In the new hibernate version, it would lead to wrong results in DB queries (potentially a bug in the new version), so it's better to get rid of it